tests/internal/test_serverless.py

```diff
@@ -79,8 +78,6 @@ def test_not_azure_function_consumption_plan():
     assert in_azure_function() is False
 
 
-# DEV: Run this test in a subprocess to avoid messing with global sys.modules state
```

Comment:
is this still not true?

Reply:
I removed the subprocess marker, so it no longer runs in a subprocess, making the comment irrelevant

Reply:
yeah, but do we need it? the comment says to avoid polluting `sys.modules` with the imports, but it is ok that we are polluting `sys.modules`?